argus/remote_client.py

```diff
@@ -66,12 +66,13 @@ def _run_command(protocol_client, shell_id, command):
             stdout, stderr, exit_code = protocol_client.get_command_output(
                 shell_id, command_id)
             if exit_code:
+                output = "\n\n".join([out for out in (stdout, stderr) if out])
```

Comment:
There are some cases, where the standard output is empty, but the standard error explains the issue for which the command exited with a positive status.